src/main/resources/db/migration/V22__device_update_assignments.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE `device_update_assignments` (
```

Comment:
Why do we need both device update assignments and ecu update assignments? device assignments could be inferred from ecu assignments?

Reply:
Device update assignment has correlationId per update and multiple Ecu update assignment has target per ecu. Can try simplify later.